lib/dice/die_cup.rb

```diff
@@ -1,26 +1,31 @@
 # frozen_string_literal: true
 
-# die_cup.rb
-
-# represents the die cup
 class DieCup
+  attr_reader :dice
+
   def initialize
     @dice = []
   end
 
-  def add_die(die)
-    @dice.push die
+  def loaded?
+    @dice.any?
   end
 
-  def remove_die(die)
-    @dice.delete die
+  # TODO: validate arguments are Die objects
+  def add_dice(*dice)
+    dice.each { |die| @dice << die }
+  end
+
+  def remove_dice(*dice)
+    dice.each { |die| @dice.delete die }
   end
 
   def roll
-    rolled_value = 0
-    @dice.each do |die|
-      rolled_value += die.roll
+    unless @dice.empty?
```

Comment:
should this method return 0 if the dice cup in empty?  the roll method should always return a number

Reply:
Awesome, I'll make those changes.